framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -422,7 +422,7 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
         this.stopMomentum(this.__momentum[domEvent.pointerId]);
       }
       // do nothing if we don't need to scroll
-      if ((Math.abs(deltaY) < 1 && Math.abs(deltaX) < 1) || this.__stopMomentum[oldTimeoutId]) {
+      if ((Math.abs(deltaY) < 1 && Math.abs(deltaX) < 1) || this.__stopMomentum[oldTimeoutId] || !this.__window) {
         delete this.__stopMomentum[oldTimeoutId];
```

Comment:
It is declared in `qx.event.handler.Gesture` which is a subclass of `qx.event.handler.GestureCore`, but as I am writing this I wonder if it is possible to access a private member from a subclass?

If not I guess `__window` needs to be changed to `_window`.